README.md

```diff
@@ -0,0 +1,25 @@
+strong-pack
+===========
+
+Simple wrapper for `tar` for packaging a node application without ignoring
+anything except VCS files (`.git`, `.svn`, `.hg`, etc.).
+
+### Usage
+
+The following example is similar to running `npm pack path/to/app` if npm
+ignored all of the normal rules for ignoring/including files.
+
+```js
+var fs = require('fs');
+var gz = require('zlib').Gzip();
+var pack = require('strong-pack');
+var tgz = fs.createWriteStream('app.tgz', 'binary');
+pack('path/to/app').pipe(gz).pipe(tgz);
```

Comment:
@pthieu You may prefer to use https://www.npmjs.com/package/pump, it avoids some error-prone error-handling.